lib/cirro_io_v2/resources/gig.rb

```diff
@@ -0,0 +1,18 @@
+module CirroIOV2
+  module Resources
+    class Gig < Base
+      def create(title:, description:, url:, start_at:, end_at:, total_seats:, invitation_mode:, filter_query:,
```

Comment:
how about passing in `params = {}`? it's more flexible.
there's no need to validate anything in the gem the API will take care of validating params anyway.